### PR TITLE
Add DIGEST_LENGTH in README for SHA-2 and HMAC.

### PR DESCRIPTION
**Pull request by andreistefanescu**
Clarify the distinction between BLOCK_LENGTH and DIGEST_LENGTH in README
for SHA-2 and HMAC.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: SAW/README.md [hunk 1]
```diff
@@ -28,26 +28,26 @@ The verification ensures that each verified function has the following general p
 
 ### SHA-2
 
-The EVP_Digest* functions are verified to have the following properties related to SHA-2. For more detailed specifications, see [evp-function-specs.saw](proof/SHA512/evp-function-specs.saw). BLOCK_LENGTH is the block length of the hash function, in bytes. For example, BLOCK_LENGTH for SHA-384 is 48.
+The EVP_Digest* functions are verified to have the following properties related to SHA-2. For more detailed specifications, see [evp-function-specs.saw](proof/SHA512/evp-function-specs.saw). BLOCK_LENGTH is the block length of the hash function, in bytes. DIGEST_LENGTH is the digest length of the hash function, in bytes. For example, for SHA-384, BLOCK_LENGTH is 64 and DIGEST_LENGTH is 48.
```
Comment (apetcher-amazon) [inline SAW/README.md:31]: It doesn't look like BLOCK_LENGTH is used in the text below. Can you remove the sentence that defines it?

**Review by apetcher-amazon — COMMENTED** [review]
Thanks for catching this.

File: SAW/README.md (hunk 1 above)
Comment (andreistefanescu) [reply SAW/README.md:31]: `BLOCK_LENGTH` is used in the text for `EVP_DigestUpdate`: `(n+k)%BLOCK_LENGTH`

Comment (apetcher-amazon) [reply SAW/README.md:31]: You're right. I missed it. Thanks.

**Review by apetcher-amazon — APPROVED** [review]

**Pull request merged**